plasticos_documents_native/models/document_sync.py

```
@@ -0,0 +1,172 @@
+    DOC_TYPE_TAG_MAP = {
+        "bol": "BOL",
+        "invoice": "INV",
+        "scale_ticket": "SCALE",
+        "contract": "CONTRACT",
+        "certification": "CERT",
+        "permit": "PERMIT",
+        "lab_report": "LAB",
+        "photo": "PHOTO",
+        "correspondence": "CORR",
+        "other": "OTHER",
+    }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)